photo-service/src/queueProducer/rabbitmqProducer.ts

```diff
@@ -0,0 +1,30 @@
+import {Connection, Channel, connect} from "amqplib";
+import IQueueProducer from "./IQueueProducer";
+import IMessageToUploadPhotos from "./IMessageToUploadPhotos";
+
+export default class RabbitmqProducer implements IQueueProducer {
+    private connection: Connection | any;
```

Comment:
Can we somehow avoid explicit `any` usage?

Reply:
Makes sense. Fixed.